ansible/roles-infra/infra-osp-create-inventory/tasks/main.yml

```diff
@@ -61,6 +56,22 @@
     - create_inventory
     - must
 
+  # This should be used if you have VMs with multiple NICs.
+  # Make sure you have multi_network_primary variable set indicating the
+  # name of the network you want to pull the IP from
+  - name: Make sure private IP is set if using multiple networks
+    when: multi_network | default(false)
```

Comment:
Does this need a `| bool` at the end?

Reply:
if multi_network is a string, like "false", it gives the following warning:

```
[DEPRECATION WARNING]: evaluating 'f2' as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the 
future. Also see CONDITIONAL_BARE_VARS configuration toggle. This feature will be removed in version 2.12. Deprecation warnings can be disabled by
 setting deprecation_warnings=False in ansible.cfg.```

So, yes, i think we want `|bool`